lib/util/errHandler.js

```diff
@@ -22,4 +22,7 @@ var errHandler = function(err) {
   }
 };
 
+/**
+ * Decides how to handle an error, then responds accordingly.
+ */
 module.exports = errHandler;
```

Comment:
If module.exports is only the one function, move it here and put the function documentation above the exports statement.